src/nodes/Image.tsx

```diff
@@ -226,51 +253,92 @@ export default class Image extends Node {
   };
 
   toMarkdown(state, node) {
-    state.write(
+    let markdown =
       "![" +
-        state.esc((node.attrs.alt || "").replace("\n", "") || "") +
-        "](" +
-        state.esc(node.attrs.src) +
-        ")"
-    );
+      state.esc((node.attrs.alt || "").replace("\n", "") || "") +
+      "](" +
+      state.esc(node.attrs.src);
+    if (node.attrs.layoutClass) {
+      markdown += ' "' + state.esc(node.attrs.layoutClass) + '"';
+    }
+    if (node.attrs.title) {
```

Comment:
Should we consider if both exist? Perhaps have `layoutClass` take precedence over title with an `else`?